pkg/schedule/config/config.go

```diff
@@ -67,6 +67,9 @@ const (
 	defaultRegionScoreFormulaVersion = "v2"
 	defaultLeaderSchedulePolicy      = "count"
 	defaultStoreLimitVersion         = "v1"
+	defaultPatrolRegionWorkerCount   = 1
+	maxPatrolRegionWorkerCount       = 8
```

Comment:
Maybe it too small and not be changed？how about using the core num as the max limit？

Reply:
Current tests show that 8 is enough, if needed in the future I think it can be increased or core num can be used.

Reply:
OK